### PR TITLE
STENCIL-3062 Fixing tests

### PR DESCRIPTION
**Pull request by mcampa**
@mjschock

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,6 @@ language: node_js
 sudo: false
 node_js:
   - 4
-  - 6
```
Comment (mjschock) [inline .travis.yml]: are we dropping support for node 6 because we'll be splitting out the functionality?

**Comment by mjschock** [comment]
LGTM 👍  - just wondering about the node 6 support drop

File: .travis.yml (hunk 1 above)
Comment (mcampa) [inline .travis.yml]: node-sass does not work on node 6. I added [here](https://github.com/bigcommerce/stencil-styles/pull/21) by mistake

Comment (mjschock) [inline .travis.yml]: got it

**Pull request merged**